crates/router/src/core/payment_methods/cards.rs

```diff
@@ -1324,7 +1324,10 @@ impl BasiliskCardSupport {
             .change_context(errors::ApiErrorResponse::InternalServerError)
             .attach_printable("Wrapped value2 construction failed when saving card to locker")?;
 
-        vault::create_tokenize(state, value1, Some(value2), payment_token.to_string()).await?;
+        let lookup_key =
+            vault::create_tokenize(state, value1, Some(value2), payment_token.to_string()).await?;
+        vault::add_tokenize_data_task(&*state.store, &lookup_key, "Delete_Tokenize_Data_Workflow")
```

Comment:
this will be `delete_tokenize_data_take` right?

Reply:
add is for adding in pt for deletion

Reply:
`add_tokenize_data_task` not giving the meaning of the function.

`add_delete_tokenized_data_task`  gives the meaning we are delete too

Reply:
maybe something like `schedule_delete_tokenize_task`?

Reply:
sure